.github/workflows/release.yml

```diff
@@ -29,9 +29,12 @@ jobs:
         run: yarn install --cwd=release
 
       - name: Release
-        run: ./release/node_modules/.bin/semantic-release --extends ./release/.releaserc.js
+        with:
+          appId: ${{ vars.APP_ID }}
```

Comment:
It wouldn't work this way, you need to replace the whole workflow to reusable one https://github.com/fingerprintjs/dx-team-toolkit?tab=readme-ov-file#6-release-server-sdk

Reply:
Steps:
- `./gradlew jar --no-daemon`
- `./gradlew publishToMavenLocal`

We need to move to semantic release side using ` "@semantic-release/exec"` plugin